openedx/core/djangoapps/content/course_structures/tasks.py

```diff
@@ -0,0 +1,79 @@
+import json
+import logging
+
+from celery.task import task
+from opaque_keys.edx.keys import CourseKey
+from xmodule.modulestore.django import modulestore
+
+
+log = logging.getLogger(__name__)
```

Comment:
Okay, I see it's even more complicated, with non-task code calling the task-annotated code directly (e.g. for the management command).  As-is, however, I am concerned that no log messages will appear in the worker log files from this.  I'm assuming that using this name for the logger is not too much of a problem -- the logging should show up on worker machines as well as LMS boxes.